src/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -1,23 +1,64 @@
+/* eslint-disable unicorn/prevent-abbreviations */
```

Comment:
Necessary? I'm not sure which plugin it is but my VS Code automatically updates names such as `_args` for me upon save.  For consistency, I think this should stay enabled. 
```suggestion
```